index.html

```diff
@@ -0,0 +1,782 @@
+                                    </path>
+                                </svg>
+                                <b>Cooking</b>
+                            </a>
+                        </li>
+                    </ul>
+                    <ul class="subscription-list">
+                        <li><a class="swap-text-decoration" href="#">Email Newsletters</a></li>
+                        <li><a class="swap-text-decoration" href="#">Corporate Subscriptions</a></li>
+                        <li><a class="swap-text-decoration" href="#">Education Rate</a></li>
+                    </ul>
+                    <ul>
+                        <li><a class="swap-text-decoration" href="#">Mobile Applications</a></li>
+                        <li><a class="swap-text-decoration" href="#">Replica Edition</a></li>
+                        <li><a class="swap-text-decoration" href="#">International</a></li>
+                        <li><a class="swap-text-decoration" href="#">Canada</a></li>
+                        <li><a class="swap-text-decoration" href="#">Español</a></li>
+                        <li><a class="swap-text-decoration" href="#">中文网</a></li>
+                    </ul>
+                </section>
+            </nav>
+        </div>
+    </article>
+
+    <footer class="width-1200">
+        <nav>
+            <ul>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">© 2021 Made by Maha Magdy</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">NYTCo</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Contact Us</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Accessibility</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Work with us</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Advertise</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">T Brand Studio</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Your Ad Choices</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Privacy Policy</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Terms of Service</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Terms of Sale</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Site Map</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Help</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Subscriptions</a></li>
+            </ul>
+        </nav>
```

Comment:
## Implement missing sections
Implement the following missing sections 👇 
- [ ] `Suggested newsletters for you` Section.
- [ ] `Most Popular` Section.
- [ ] `More in Space and Astronomy` Section.
- [ ] `Editors’ Picks` Section - That is beneath the `More in Space and Astronomy` Section.
- [ ] The sticky bottom footer just as it appears in the [Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html). 
- ![footer](https://user-images.githubusercontent.com/22345961/108410085-06a2ce00-7238-11eb-8b54-5e4278de1009.png)